lib/build.api.js

```diff
@@ -10,6 +10,8 @@ const _ = require('lodash');
 const fs = require('fs-extra');
 const del = require('del');
 const path = require('path');
+const glob = require("glob");
+const fx = require('mkdir-recursive');
```

Comment:
No need for this, fs-extra mkdirSync provides recursive mkdirs: https://github.com/jprichardson/node-fs-extra#mkdirsdir-callback